### ISSUE TITLE
[Bug]: ControlNet doesn't not guide the model at the hires.fix stage

### ISSUE THREAD

**Issue by zpdDG4gta8XKpMCd** [Mikubill/sd-webui-controlnet#670]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

I donwloaded the latest webui and sd-webui-controlnet.

So the low-res images comes out exactly guided by the ControlNet without any problems. I like it and I wish it used by the highres.fix upscaler.

But then at the hires.fix stage
- if denoising is  `0.5` or `0.6` the image gets denoised to beyond repair so all ControlNet guidance is lost
- f denoising is set `0.3` or `0.4` although it resembles the low-res image, the image doesn't converge through 20 sampling steps and stays mostly denoised (unfinished), I don't know why it is that.

But regardless, in either case, with high or low denoising, the ControlNet doesn't do anything at the highres.fix stage, so I cannot get the upscaled image closely follow the low-res image. So ControlNet turns out to be useless with highres.fix.

Is it supposed to work like this?

### Steps to reproduce the problem

1. Use the standard SD1.5 model: `v1-5-pruned-emaonly.ckpt`
2. Make a random `512 x 512` image that is going to be used as a source image (see the exact settings and image in the screenshot, in my case the prompt is `Bunny`, no negative prompt, seed `123`)

![image](https://user-images.githubusercontent.com/937933/228041740-92f6a316-e48c-43ec-9dd5-d5693c500ae1.png)

3. Set the highres.fix as follows, with denoising strength set at `0.6`

![image](https://user-images.githubusercontent.com/937933/228042394-90f22f3d-a262-4262-b0ea-435dd06a8445.png)

4. Enabled the ControlNet using the image from step 2, canny adapter and canny model, as follows
![image](https://user-images.githubusercontent.com/937933/228042701-24367466-81f0-4df5-abd4-77688bd4e3dc.png)

5. Generate

6. Observe weird upscale artifacts, due to denoising and no guidance from ControlNet after
![image](https://user-images.githubusercontent.com/937933/228043121-fca5c896-8a43-44ce-a6b1-4af83ea17435.png)

7. Set Denoising strength to `0.3`

8. Generate

9. Observe the image didn't converge (aka unfinished, blurry) and no guidance from ControlNet
![image](https://user-images.githubusercontent.com/937933/228043974-c9f24844-e173-4f36-8ffb-707fb78729f9.png)

### What should have happened?

I expect the ControlNet guidance at the hires.fix stage, but instead the low-res image gets denoised beyond recognition and ControlNet doesn't kick in.

### Commit where the problem happens

webui: 4c1ad743e3baf1246db0711aa0107debf036a12b
controlnet: 241c05f8c9d3c5abe637187e3c4bb46f17447029

### What browsers do you use to access the UI ?

```
Google Chrome | 111.0.5563.111 (Official Build) (64-bit) (cohort: Stable)
Revision | 3da375c9e9e5ad9b556af5df708a6e6c01d8cfb1-refs/branch-heads/5563@{#1202}
OS | Windows 10 Version 21H2 (Build 19044.2728)
JavaScript | V8 11.1.277.17
User Agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36
Active Variations | 5e3a236d-4113a79e
```

### Command Line Arguments

```Shell
set COMMANDLINE_ARGS=--precision full --no-half
```

### Console logs

```Shell
venv "D:\sd\stable-diffusion-webui\venv\Scripts\Python.exe"
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Commit hash: a0fbdfcb958b927a3205d205a673f0ee21d74431
Installing requirements for Web UI

Installing requirements for Unprompted - img2pez
Installing requirements for Unprompted - pix2pix_zero

Launching Web UI with arguments: --precision full --no-half
No module 'xformers'. Proceeding without it.
Loading Unprompted v7.9.1 by Therefore Games
(SETUP) Initializing Unprompted object...
(SETUP) Loading configuration files...
(SETUP) Debug mode is False
Loading weights [cc6cb27103] from D:\sd\stable-diffusion-webui\models\Stable-diffusion\v1-5-pruned-emaonly.ckpt
Creating model from config: D:\sd\stable-diffusion-webui\configs\v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Applying cross attention optimization (Doggettx).
Textual inversion embeddings loaded(0):
Model loaded in 5.4s (load weights from disk: 1.3s, create model: 0.4s, apply weights to model: 0.5s, move model to device: 2.2s, load textual inversion embeddings: 1.0s).
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 15.0s (import torch: 2.7s, import gradio: 1.7s, import ldm: 0.3s, other imports: 1.2s, list extensions: 0.2s, setup codeformer: 0.1s, load scripts: 2.0s, load SD checkpoint: 5.6s, create ui: 0.8s, gradio launch: 0.3s).

100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:07<00:00,  2.76it/s]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 20/20 [00:04<00:00,  4.34it/s]
Total progress: 100%|█████████████Loading model: control_canny-fp16 [e3fe7712]█████████| 20/20 [00:04<00:00,  6.11it/s]
Loaded state_dict from [D:\sd\stable-diffusion-webui\models\ControlNet\control_canny-fp16.safetensors]
ControlNet model control_canny-fp16 [e3fe7712] loaded.
Loading preprocessor: canny
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:05<00:00,  3.52it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:31<00:00,  1.59s/it]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 40/40 [05:12<00:00,  7.82s/it]
Loading model from cache: control_canny-fp16 [e3fe7712]████████████████████████████████| 40/40 [05:12<00:00,  1.54s/it]
Loading preprocessor: canny
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:05<00:00,  3.56it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:32<00:00,  1.60s/it]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 40/40 [00:39<00:00,  1.01it/s]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 40/40 [00:39<00:00,  1.64s/it]
```

**Comment by ljleb** [comment]
Check the option "Use mid-control on highres pass (second pass)" in settings.

**Comment by zpdDG4gta8XKpMCd** [comment]
I did, by default this option is on. Turning it on/off doesn't seem do do anything.

**Comment by GretaThuumberg** [comment]
Same problem here. If I use either hires fix or a higher resolution it completely ignores the Controlnet input. E.g. Openpose spawns extra arms and in general creates a massive mess of elements and limbs.
